hw/top_earlgrey/data/ip/chip_alert_handler_testplan.hjson

```diff
@@ -0,0 +1,268 @@
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_alert_test"]
+    }
+    {
+      name: chip_sw_alert_handler_escalations
+      desc: '''Verify all alert escalation paths.
+
+            Verify all escalation paths triggered by an alert.
+            - Verify the first escalation results in NMI interrupt serviced by the CPU.
+            - Verify the second results in device being put in scrap state, via the LC JTAG TAP.
+            - Verify the third results in chip reset.
+            - Ensure that all escalation handshakes complete without errors.
+              - This cannot be checked in sival
+            '''
+      features: ["ALERT_HANDLER.ALERT.ESCALATE",
+                 "ALERT_HANDLER.ESCALATION.PHASES"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_alert_handler_escalation"]
+    }
+    {
+      name: chip_sw_alert_handler_escalation_methods
+      desc: '''Verify the different escalation trigger methods
+
+            For each escalation class, verify the escalation occurs (just the first stage, NMI
+            escalation) with all of the possible trigger methods.
+            - Escalation triggered when the number of alerts seen in the class is greater than the
+              threshold
+            - Escalation triggered when initial interrupt isn't acknowledged within the timeout
+            '''
+      features: ["ALERT_HANDLER.ESCALATION.COUNT",
+                 "ALERT_HANDLER.ESCALATION.TIMEOUT"]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: []
+    }
+    {
+      name: chip_sw_all_escalation_resets
+      desc: '''Verify escalation from all unit integrity errors.
+
+            Inject integrity errors in any unit that has a one-hot checker for CSR register
+            writes, and verify escalation is triggered. Allow escalation to go through reset.
+            Use the rstmgr alert info and the unit's fault CSRs to check the alert cause is right.
+            Each run of the test randomly chooses some one-hot checker for the error to be injected.
+            Keep state across resets in flash to check the expected interrupts and the right number
+            of resets occur.
+            - Verify the integrity error results in a regular interrupt.
+            - Verify the first escalation results in NMI serviced by the CPU.
+            - Verify the alert id in both these interrupts.
+            - Verify the unit's fault CSR correctly captured the fault kind.
+            - Verify any timer interrupts are disabled by escalation.
+            - Verify after the escalation reset all faults are cleared, and that the alert
+              info captured the correct alert.
+            - Check that no additional resets occur.
+            '''
+      stage: V2
+      si_stage: None
```

Comment:
Some of these checks are needed, like timer interrupts being disabled, faults being cleared, and alert info. Perhaps the other tests that escalate will perform these checks, but it may be good to mention these checks in these other tests.